gen/templates/model.go

```diff
@@ -135,6 +135,9 @@ func (data {{camelCase .Name}}) getPath() string {
 
 func (data {{camelCase .Name}}) toBody(ctx context.Context, state {{camelCase .Name}}) string {
 	body := ""
+	if data.Id.ValueString() != "" {
```

Comment:
I think I have removed this in one of the recent commits to main, as we don't need to include the ID in the payload.

Reply:
removed